pykickstart/commands/zipl.py

```diff
@@ -0,0 +1,90 @@
+#
+# Vendula Poncova <vponcova@redhat.com>
```

Comment:
There's no need for names in the copyright statement, git log and blame work fine for tracking down who wrote something.

Reply:
Fixed.